js/handlebarsHelpers.js

```diff
@@ -4,4 +4,7 @@ Handlebars.registerHelper('arr_empty', function(arr, opts) {
     } else {
         return opts.inverse(this);
     }
+});
+Handlebars.registerHelper('tel', function(phone) {
+    return phone.replace(/\D/g, '').replace(/^1?/, '+1');
```

Comment:
I like the idea of making this a helper. We may need to expand this to output a more common final format like (xxx) xxx-xxxx or x-xxx-xxx-xxxx

Reply:
@jonjanelle I like that idea, but wasn't too confident with being too bold about the phone numbers without checking out the source data first. This seemed to be okay for the data that I was seeing; is there a way to enforce a rule in the source spreadsheet so that all telephone numbers have the same number of digits or formatting?